token-lending/program/src/state/reserve.rs

```diff
@@ -356,14 +355,15 @@ pub struct ReserveLiquidity {
     pub supply_pubkey: Pubkey,
     /// Reserve liquidity fee receiver address
     pub fee_receiver: Pubkey,
-    /// Optional reserve liquidity oracle state account
-    pub oracle_pubkey: COption<Pubkey>,
+    /// Reserve liquidity oracle account
+    pub oracle_pubkey: Pubkey,
     /// Reserve liquidity available
     pub available_amount: u64,
     /// Reserve liquidity borrowed
     pub borrowed_amount_wads: Decimal,
     /// Reserve liquidity cumulative borrow rate
     pub cumulative_borrow_rate_wads: Decimal,
+    // @TODO: make Decimal
```

Comment:
I'm considering making the market price here a Decimal, as it's used on all the obligation refreshes and would require fewer math operations.

Reply:
Done.